pycqed/analysis/measurement_analysis.py

```diff
@@ -455,24 +557,86 @@ def get_naming_and_values(self):
             raise ValueError('datasaving_format "%s " not recognized'
                              % datasaving_format)
 
-    def plot_results_vs_sweepparam(self, x, y, fig, ax, show=False,
-                                   marker='-o',
-                                   xlabel=None, x_unit=None,
-                                   ylabel=None, y_unit=None,
-                                   log=False, label=None, **kw):
-        save = kw.pop('save', False)
-        self.plot_title = kw.pop('plot_title',
-                                 textwrap.fill(self.timestamp_string + '_' +
-                                               self.measurementstring, 40))
+    def scale_sweep_points(self, unit_prefix = '', TwoD=False):
```

Comment:
Why is there a scale sweep points method here? 
I think there should be one and only one such method (the one used in set_xlabel/set_ylabel) that you are also importing in this module. 

Let's avoid multiple functions that do the same thing and the related problems.